src/pages/en/guides/content-collections.mdx

```diff
@@ -16,166 +16,152 @@ import TypeScriptSettingTabs from '~/components/tabs/TypeScriptSettingTabs.astro
-
-<TypeScriptSettingTabs>
-  <Fragment slot="relaxed">
-  If you are extending Astro's `base` TypeScript config (e.g. you chose "relaxed" TypeScript when creating your project with `create astro`), add `"strictNullChecks": true` under `compilerOptions`.
-  
-  Or, you can change your `base` TypeScript config to `strict` or `strictest` to use a broader set of stricter type-checks in your project.
-
-  ```json title="tsconfig.json" ins={3-5} "base"
-  {
-    "extends": "astro/tsconfigs/base",
-    "compilerOptions": {
-      "strictNullChecks": true
-    }
-  }
-  ```
-  </Fragment>
-  <Fragment slot="strict">
-  No update necessary! 
-  
-  `"strictNullChecks": true` is already enabled in your `strict` or `strictest` configuration.
-
-  ```json title="tsconfig.json" "strict" "strictest"
-  {
-    "extends": "astro/tsconfigs/strict" // or `strictest`
-  }
-  ```
-  </Fragment>
-  <Fragment slot="custom">
-  If you have a custom TypeScript configuration that does not include `strict: true`, add `"strictNullChecks": true` under `compilerOptions` if it does not already exist.
-
-  ```json title="tsconfig.json" ins={3}
-  {
-    "compilerOptions": {
-      "strictNullChecks": true
-    }
-  }
-  ```
-  </Fragment>
-</TypeScriptSettingTabs>
-
-### Set up the `.astro` directory
-
-Astro generates TypeScript types from your content collections in the `.astro` directory. These types will be updated anytime you run the [`astro dev`](/en/reference/cli-reference/#astro-dev), [`astro build`](/en/reference/cli-reference/#astro-build), or [`astro sync`](/en/reference/cli-reference/#astro-sync) commands.
-
-:::tip
-If you're using Git for version control, we recommend ignoring this generated directory by adding `.astro` to your `.gitignore`.
-:::
-
-These commands will also generate a [reference path](https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html#-reference-path-) to include `.astro` types in your project. A [`src/env.d.ts` file](/en/guides/typescript/#setup) will be created for you if one does not exist, and the following will be added:
-
-```ts title="src/env.d.ts"
-// using a relative path `../` to the `.astro` directory
-/// <reference types="../.astro/types.d.ts" />
-```
-
-## The content directory
-
-Astro treats the `src/content/` directory as special. This is where **collections** (folders) of Markdown/MDX **entries** (files) can be stored, with a single configuration file to define each collection's **schema** (frontmatter data types and shape). Files other than your `.md`/`.mdx` content are not permitted inside `src/content/`.
+<FileTree>
+- src/content/
+  - **newsletter/** 
+    - week-1.md
+    - week-2.md
+    - week-3.md
+</FileTree>
 
-## Collections
+Once you have a content collection in place with at least one content entry, you can query your content using Astro's built-in APIs.
```

Comment:
you're right. Leaving it in but making it link to the query APIs, in case someone wants to skip over the configuration section and go straight to usage.